### DISCUSSION TITLE
amend the Wiki documentary - install on Debian 11.6 Bullseye

### DISCUSSION THREAD

**Discussion by Zweihorn** [pawelsalawa/sqlitestudio#5430] [General]
### Details - Amend the Wiki documentary
_Just adding_ some **quick compile & install** documentary for installing `sqlitestudio-3.4.1` on **Linux - Debian 11.6 Bullseye**

### Steps to reproduce
Please find below the documentary amend to reproduce a straight forward **quick compile & install** for _Linux - Debian 11.6 Bullseye (stable)_ which aims at **Wiki amendment** to [Manual compilation - more detailed description](https://github.com/pawelsalawa/sqlitestudio/wiki/Instructions_for_compilation_under_Linux#manual-compilation---more-detailed-description) supposedly.

### Operating system
Debian 11.6 Bullseye (stable) Linux amd64

### SQLiteStudio version
sqlitestudio-3.4.1

###

### Quick compile & install with Debian 11.6 Bullseye

**Wiki amendment** under the new section '**Quick compile & install with Debian 11.6 Bullseye (stable)'** supposedly.
For amendment to you Wiki at [Manual compilation - more detailed description](https://github.com/pawelsalawa/sqlitestudio/wiki/Instructions_for_compilation_under_Linux#manual-compilation---more-detailed-description) at your consideration.

```
### --- Debian 11.6 Bullseye ---

$ sudo apt install g++

$ sudo apt install qtcreator
$ sudo apt install qtdeclarative5-dev
$ sudo apt install qttools5-dev
$ sudo apt install libqt5svg5-dev

$ sudo apt install libreadline-dev
$ sudo apt install tcl-dev

$ sudo apt install python3-dev
$ sudo apt install libssl-dev

$ mkdir -p /home/user/projects/sqlitestudio

$ cd /home/user/projects/sqlitestudio

$ wget https://github.com/pawelsalawa/sqlitestudio/archive/refs/tags/3.4.1.tar.gz
$ tar xf 3.4.1.tar.gz

$ cd sqlitestudio-3.4.1

$ mkdir -p output/build
$ cd output/build

$ qmake ../../SQLiteStudio3
$ make

$ mkdir Plugins
$ cd Plugins

$ qmake ../../../Plugins "INCLUDEPATH += /usr/include/python3.9"
$ make

$ make install

### done.
```

NOTE: Was amended to include 'Plugins' make and additional required installs.
Hope this helps.

🌻

**Comment by Zweihorn** [comment]
Based on my little experiment in reply the topic [Re: How do database files look like in Minetest](https://forum.minetest.net/viewtopic.php?p=420279#p420279) in the [Minetest](https://www.minetest.net/get-involved/) forum recently.

Furthermore, I have my little role as the new maintainer to t[he minetest port at MacPorts](https://ports.macports.org/port/minetest/).

**Comment by pawelsalawa** [comment]
I think you are missing Plugins in this build recipe, aren't you?
```
qmake ../../Plugins
make
```

**Comment by Zweihorn** [comment]
[dzięki](https://dict.leo.org/polnisch-deutsch/dzi%C4%99ki) [twojej](https://dict.leo.org/polnisch-deutsch/twojej) [pomocy](https://dict.leo.org/polnisch-deutsch/pomocy)

> I think you are missing Plugins in this build recipe, aren't you?
> 
> ```
> qmake ../../Plugins
> make
> ```

Hmm, your lucky day apparently today. However, to be correct use `qmake ../../../Plugins` with three `../` please.

The above script **code was amended** to include 'Plugins' make and **additional required installs**. 

CAVEAT: I did not do the 'make install' (to be transparent on that) and I have no interest to do that in my VirtualBox Debian dev environment on my Mac.

I would call it a day. Lesson learned.

@pawelsalawa - THX - Keep on the good work with your SQLiteStudio3 tool and propser.
